pkg/eval/builtin_fn_fs_test.go

```diff
@@ -23,3 +32,36 @@ func TestBuiltinFnFS(t *testing.T) {
 		That(`-is-dir ~/file`).Puts(false),
 	)
 }
+
+func TestBuiltinCd(t *testing.T) {
```

Comment:
This test has quite a bit of setup, but not every test case uses those setups. It'd be cleaner to break up this test into multiple independent tests.

Reply:
That seems counterproductive. The setup is done once -- not per test. If the tests are split up then any shared setup, such as creating a temp dir, gets done more than once. If there were a large number of individual tests it might be worthwhile from a clarity viewpoint, but there are just a handful of tests.